core/src/main/java/com/klaytn/caver/contract/Contract.java

```diff
@@ -205,6 +207,198 @@ public KlayLogs getPastEvent(String eventName, KlayLogFilter filterOption) throw
+     * It is recommended to use this function when you want to execute one of the functions with the same number of parameters.
+     * When creating CallObject, it need not to fill 'data', 'to' fields.
+     * The 'data', 'to' fields automatically filled in call() method.
+     * @param methodName The smart contract method name to execute.
+     * @param callObject A CallObject instance to 'call' smart contract method.
+     * @param methodArguments The arguments that need to execute smart contract method.
+     * @return List
+     * @throws IOException
+     * @throws ClassNotFoundException
+     */
+    public List<Type> callWithSolidityWrapper(String methodName, CallObject callObject, Type... methodArguments) throws IOException, ClassNotFoundException {
+        ContractMethod contractMethod = this.getMethod(methodName);
+
+        return contractMethod.callWithSolidityWrapper(Arrays.asList(methodArguments), callObject);
+    }
+
+    /**
+     * Send a transaction to smart contract and execute its method.
+     * It is used defaultSendOption field to sendOptions.
+     * It sets TransactionReceiptProcessor to PollingTransactionReceiptProcessor.
+     * @param methodName The smart contract method name to execute
+     * @param methodArguments The arguments that need to execute smart contract method.
+     * @return TransactionReceiptData
+     * @throws NoSuchMethodException
+     * @throws IOException
+     * @throws InstantiationException
+     * @throws ClassNotFoundException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws TransactionException
+     */
+    public TransactionReceipt.TransactionReceiptData send(String methodName, Object... methodArguments) throws NoSuchMethodException, IOException, InstantiationException, ClassNotFoundException, IllegalAccessException, InvocationTargetException, TransactionException {
+        return send(methodName, null, methodArguments);
+    }
+
+    /**
+     * Send a transaction to smart contract and execute its method.
+     * It sets TransactionReceiptProcessor to PollingTransactionReceiptProcessor.
+     * @param methodName The smart contract method name to execute
+     * @param options An option to execute smart contract method.
+     * @param methodArguments The arguments that need to execute smart contract method.
+     * @return TransactionReceiptData
+     * @throws NoSuchMethodException
+     * @throws TransactionException
+     * @throws IOException
+     * @throws InstantiationException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws ClassNotFoundException
+     */
+    public TransactionReceipt.TransactionReceiptData send(String methodName, SendOptions options, Object... methodArguments) throws NoSuchMethodException, TransactionException, IOException, InstantiationException, IllegalAccessException, InvocationTargetException, ClassNotFoundException {
```

Comment:
Should ʻObject...methodArguments` be passed as the last parameter unconditionally?